src/bin/hal/cmd/tx.rs

```diff
@@ -195,7 +195,7 @@ pub fn create_transaction(info: TransactionInfo) -> Transaction {
 
 	Transaction {
 		version: info.version.expect("Field \"version\" is required."),
-		lock_time: info.locktime.expect("Field \"locktime\" is required."),
+		lock_time: bitcoin::LockTime::from_consensus(info.locktime.expect("Field \"locktime\" is required.")).into(),
```

Comment:
Change this to "from_height" (or the other from at that version of the lib) and handle the error case.